### PR TITLE
modified your comments Sara

### PR DESCRIPTION
**Pull request by Knulpinette**

### CODE REVIEW

File: srcs/parsing/parser.c [hunk 1]
```diff
@@ -160,7 +160,10 @@ void		assign_tokens(t_cmd_table *cmd_table)
 			cmd_table->redirs[i].arg = remove_quotes(cmd_table->redirs[i].arg);
 		i++;
 	}
-	cmd_table->cmd_name = cmd_table->tokens[0].text;
+	if (cmd_table->tokens[0].text)
+		cmd_table->cmd_name = cmd_table->tokens[0].text;
+	else
+		cmd_table->cmd_name = ft_strdup("");
```
Comment (arieivs) [inline srcs/parsing/parser.c:166]: Why this?

**Comment by arieivs** [comment]
Ok, just added these changes in my branch.

File: srcs/utils/error_exit.c [hunk 2]
```diff
@@ -27,8 +27,6 @@ void	error_message(t_error code)
 	/* can you check the input:     > test       to see if you have leaks or a segfault ? */
 	else if (code == NO_CMD)
 		ft_putstr_fd("There is no command.\n", STDERR_FILENO);
```
Comment (arieivs) [inline srcs/utils/error_exit.c:29]: One last question, I don't think "no command" should be an error, at least in bash it doesn't give any error, or did you see it printing out an error at some point?

**Comment by Knulpinette** [comment]
Ah ! Also something I forgot to remove =) I tried that while I was
debugging why I had segfault with no cmd but that wasn't the issue ^^
You can just get rid of it !

Le jeu. 14 avr. 2022 à 09:21, Sara Vieira ***@***.***> a
écrit :

**Pull request closed without merging**